airbyte-integrations/connectors/source-outreach/source_outreach/source.py

```diff
@@ -22,21 +25,39 @@
 class OutreachStream(HttpStream, ABC):
 
     url_base = _URL_BASE
+    primary_key = "id"
+    page_size = 1000
```

Comment:
Should this be in the config as well?  I would assume that a higher page size means fewer API calls, so there is some connection to `max_api_requests`